src/MCPClient/lib/clientScripts/verify_AIP.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python2
+from __future__ import print_function
+import os
+from pprint import pformat
+import shutil
+import sys
+
+import django
+django.setup()
```

Comment:
Can you move this call inside the `if __name__ == '__main__':` block? I find it's more explicit and clear. If there was a case where someone is using this module then the caller should be responsible to initialize Django. Probably not very important, but I've been doing it this way recently.

Reply:
ok